libpermian/plugins/kickstart_test/__init__.py

```diff
@@ -237,10 +332,24 @@ def execute(self):
 
         test_to_crcs = self._map_tests_to_crcs(self.crcList)
         tests = list(test_to_crcs.keys())
-        current_results = KicstartTestBatchCurrentResults(tests)
+        current_results = KicstartTestBatchCurrentResults(tests, retry=self.retry)
         self.groupReportResult(self.crcList, Result('running', current_results=current_results))
 
-        command = self.runner_command + tests
+        command = self.runner_command
+
+        if self.platform:
+            command = command + ['--platform', self.platform]
+
+        if self.defaults_file_path:
```

Comment:
Please add a comment here that this is in fact "URLs overrides" or something like that.

Reply:
https://github.com/rhinstaller/permian/pull/14/commits/a1ec2bac503f1817cf400c65440dbac9205b6808